types/node/test/util.ts

```diff
@@ -127,8 +127,10 @@ import { readFile } from 'fs';
     const foo = () => {};
     // $ExpectType () => void
     util.deprecate(foo, 'foo() is deprecated, use bar() instead');
-    // $ExpectType <T extends Function>(fn: T, message: string) => T
+    // $ExpectType <T extends Function>(fn: T, message: string, code?: string | undefined) => T
```

Comment:
It should be changed here and below as well.

Reply:
@sokra update this comment?

Reply:
Changing it to `// $ExpectType <T extends Function>(fn: T, message: string, code?: string) => T` won't work:

``` text
Error in node
Error: C:/Repos/DefinitelyTyped/types/node/test/util.ts:131:5
ERROR: 131:5  expect  TypeScript@3.1 expected type to be:
  <T extends Function>(fn: T, message: string, code?: string) => T
got:
  <T extends Function>(fn: T, message: string, code?: string | undefined) => T
ERROR: 133:5  expect  TypeScript@3.1 expected type to be:
  <T extends Function>(fn: T, message: string, code?: string) => T
got:
  <T extends Function>(fn: T, message: string, code?: string | undefined) => T
```

I guess typescript internally adds `| undefined` for optional arguments, so the expected type need to include this too.

The type is defined without `| undefined`:

``` ts
function deprecate<T extends Function>(fn: T, message: string, code?: string): T;
```

Reply:
I just want to confirm again so there are differences between tests for `latest` and `v10`
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/38753/files#diff-2d1106145a2d470a4a929f463c4a0292R949

If that's the case then I can approve it.

Reply:
If other contributors don't have any other reviews then it's ok for me.